composer.json

```diff
@@ -7,29 +7,29 @@
         "php": ">=8.2",
         "spryker/event": "^2.4.0",
         "spryker/event-dispatcher-extension": "^1.0.0",
-        "spryker/kernel": "^3.49.0",
+        "spryker/kernel": "^3.72.0",
```

Comment:
why kernel version bump? not sure how changes in the module require newer version. Same goes for Propel module too

Reply:
versions outdated, previous CI didn't work